tests/factories/doi.js

```diff
@@ -92,9 +92,8 @@ FactoryGuy.define('geoLocation', {
 FactoryGuy.define('subject', {
   default: {
     subject: 'Clinical medicine',
-    subjectScheme: 'OECD',
-    schemeUri: 'http://www.oecd.org/science/inno' ,
-    valueUri: 'http://www.oecd.org/science/inno/38235147.pdf',
+    subjectScheme: 'Fields of Science and Technology (FOS)',
+    schemeUri: 'http://www.oecd.org/science/inno/38235147.pdf',
```

Comment:
A question of clarification on this: For this example, the valueUri referees to the URI of the term for 'Clinical Medicine'?  https://support.datacite.org/docs/schema-optional-properties-v43#6c-valueuri

Reply:
Yes. And there is no specific URL for each term in FOS.